skiko/src/jvmMain/kotlin/org/jetbrains/skiko/Convertors.kt

```diff
@@ -76,3 +77,59 @@ fun BufferedImage.toBitmap(): Bitmap {
 fun BufferedImage.toImage(): Image {
     return Image.makeFromBitmap(toBitmap())
 }
+
+fun toSkikoEvent(e: MouseEvent): SkikoPointerEvent {
+    return SkikoPointerEvent(
+        e.x.toDouble(),
+        e.y.toDouble(),
+        when(e.button) {
```

Comment:
Better use `modifiersEx`. When we use it, we can know about multiple pressed buttons.